frontend/src/scenes/me/Settings/changePasswordLogic.ts

```diff
@@ -25,7 +25,7 @@ export const changePasswordLogic = kea<changePasswordLogicType>([
                         ? 'Please enter your current password'
                         : undefined,
                 password: !password
-                    ? 'Please enter your password to continue'
```

Comment:
I think this error message is referenced in this Cypress test https://github.com/PostHog/posthog/blob/master/cypress/e2e/signup.cy.ts#L26

So that needs updating as well.

Frustrating that some useful jobs don't run on forks currently 😊

Reply:
@pauldambra It's not that simple. This message is referred in **Sign up form**. So, calling it a "new password" would be more confusing to users.  Because, to users, it's obviously a new password.